### DISCUSSION TITLE
大佬，能否增加对城通网盘的支持

### DISCUSSION THREAD

**Discussion by erosaishen** [AlistGo/alist#1935] [General]
如题，或者现有设置中能否设置城通网盘？

**Comment by ampcr** [comment]
> 如题，或者现有的设置盘中可以设置城通网？

要不是会员就算了, 文件都有时效没意思..
网盘会员的话城通网盘提供的webdav可以在alist挂载..协议限制只能走本地流量了..

**Reply by erosaishen** [reply]
谢谢！原来可以通过webdav来挂载，我试了下，成功了！
